less/styles.less

```diff
@@ -349,4 +349,8 @@ h2, h3 {
   }
 }
 
-
+.eclipsefnd-video-cover img{
```

Comment:
Is this something we should be adding to solstice-assets so it can be reused in other websites? Or do you think this is something that can only be used in the sparkplug website?

Reply:
> Is this something we should be adding to solstice-assets so it can be reused in other websites? Or do you think this is something that can only be used in the sparkplug website?

This is trying to make the image thumbnail the same ratio (widh/height) as the video thumbnails. I am not sure if we could have similar situation in other websites? (Like the iframe can't be properly used for videos from some sites other than YouTube and we need to display an image thumbnail)